lib/vmpooler/statsd.rb

```diff
@@ -0,0 +1,12 @@
+require 'rubygems' unless defined?(Gem)
+
+module Vmpooler
+  class Statsd
+    def initialize(
+      s = 'statsd',
+      port = 8125
```

Comment:
Yeah... no reason this can't be one line and please don't use single-letter variable names. Can we add some yardoc to this so we know what these values are supposed to be? Right now I don't know if `Statsd#new` is actually expecting a string or if it expects a more complex object and this is just a placeholder.